draft-xchacha-rfc-draft00.txt

```diff
@@ -98,8 +100,8 @@ Table of Contents
 
    However, a more straightforward strategy can prevent nonce misuse
    conditions in environments where a large number of messages are
-   encrypted: Accept a large enough nonce such that applications can
-   generate them randomly for each message and the probability of a
+   encrypted.  Simply use a large enough nonce such that applications
+   can generate them randomly for each message and the probability of a
```

Comment:
<bike shedding> This sentence seemed a bit long before.

Reply:
Thank you for living up to this github org's name 😆